modules/authorize/templates/authorize_403.twig

```diff
@@ -2,7 +2,11 @@
 
 {% block content %}
   <h1>{{ '{authorize:Authorize:403_header}'|trans }}</h1>
+{% if reject_msg[currentLanguage] is defined %}
+  <p>{{ reject_msg[currentLanguage]|raw }}</p>
```

Comment:
This isn't necessary. We have a function that obtains a localized string from an array containing the available translations, indexed by the language code. This is accessible from twig via the `translateFromArray` filter, so this code would be:

```twig
    {%- if reject_msg %}

    <p>{{ reject_msg|translateFromArray }}</p>
    {%- else %}

    <p>{% trans %}{authorize:Authorize:403_text}{% endtrans %}</p>
    {%- endif %}
```

A side note, also. Spacing is the hardest part of twig, IMHO. We should have good spacing / indentation in the twig templates, so that we can clearly see how the code flows, but the resulting HTML should also be properly indented. When we use twig tags, there can be space before and after them. If we have a newline right after the closure of a twig tag, that newline is removed from the output. The easiest way to control space in both the template and the resulting HTML is to use `-`. Adding it right after the begining of an opening tag (`{%`) tells twig to remove **all** the white space before, including newlines. Similarly, adding it before the end of a closing tag (`-%}`) removes all white space after it. So the simplest way for me to keep things working is to remember to add two newlines after a tag, and clear all white space before a tag that's standing on it's own line. E.g.:

```twig
    {%- if something %}

    <p>Something</p>
    {%- else %}

    <p>else</p>
    {%- endif %}
```